### ISSUE TITLE
MM errors when RealPlume, SmokeScreen and RSMP are all installed

### ISSUE THREAD

**Issue by linuxgurugamer** [CobaltWolf/Bluedog-Design-Bureau#1497]
When these three are installed with BDB, gets the following errors:

```
[LOG 20:51:48.049] Applying update Bluedog_DB/Compatibility/RealPlumes/BDB_Apollo/@PART[bluedog_Apollo_LES]:AFTER[zzRealPlume]:NEEDS[zRealPlume,SmokeScreen,!RealismOverhaul] to Bluedog_DB/Parts/Apollo/bluedog_Apollo_LES.cfg/PART[bluedog_Apollo_LES]

[ERR 20:51:48.049] Error - Cannot parse variable search when inserting new key transformName = #$../../../PLUME[running-Escape]/transformName$

[ERR 20:51:48.049] Error - Cannot parse variable search when inserting new key localRotation = #$../../../PLUME[running-Escape]/localRotation[0]$,$../../../PLUME[running-Escape]/localRotation[1]$,$../../../PLUME[running-Escape]/localRotation[2]$

[ERR 20:51:48.050] Error - Cannot parse variable search when inserting new key localPosition = #$../../../PLUME[running-Escape]/plumePosition[0]$,$../../../PLUME[running-Escape]/plumePosition[1]$,$../../../PLUME[running-Escape]/plumePosition[2]$

[ERR 20:51:48.050] Error - Cannot parse variable search when inserting new key fixedScale = #$../../../PLUME[running-Escape]/plumeScale$

[ERR 20:51:48.050] Error - Cannot parse variable search when inserting new key energy = #$../../../PLUME[running-Escape]/energy$

[ERR 20:51:48.050] Error - Cannot parse variable search when inserting new key speed = #$../../../PLUME[running-Escape]/speed$

[ERR 20:51:48.050] Error - Cannot parse variable search when inserting new key emissionMult = #$../../../PLUME[running-Escape]/emissionMult$

[ERR 20:51:48.050] Error - Cannot parse variable search when inserting new key transformName = #$../../../PLUME[running-Escape]/transformName$

[ERR 20:51:48.050] Error - Cannot parse variable search when inserting new key localRotation = #$../../../PLUME[running-Escape]/localRotation[0]$,$../../../PLUME[running-Escape]/localRotation[1]$,$../../../PLUME[running-Escape]/localRotation[2]$

[ERR 20:51:48.050] Error - Cannot parse variable search when inserting new key localPosition = #$../../../PLUME[running-Escape]/plumePosition[0]$,$../../../PLUME[running-Escape]/plumePosition[1]$,$../../../PLUME[running-Escape]/plumePosition[2]$

[ERR 20:51:48.050] Error - Cannot parse variable search when inserting new key fixedScale = #$../../../PLUME[running-Escape]/plumeScale$

[ERR 20:51:48.050] Error - Cannot parse variable search when inserting new key emissionMult = #$../../../PLUME[running-Escape]/emissionMult$

[ERR 20:51:48.050] Error - Cannot parse variable search when inserting new key volume = #$../../../PLUME[running-Escape]/plumeScale$

[ERR 20:51:48.051] Error - Cannot parse variable search when inserting new key volume = #$../../../PLUME[running-Escape]/plumeScale$
```

Solved by modifying the file:  Compatibility\RealPlumes\BDB_Apollo.cfg
line 62:
@PART[bluedog_Apollo_LES]:AFTER[zzRealPlume]:NEEDS[zRealPlume,SmokeScreen,!RealismOverhaul,!RSMP]

Note:  I'm not sure if the other stanzas in this file are compatible with RSMP, this is the change that stopped the errors.

If I get more info, I'll post

**Comment by Rodg88** [comment]
It does seem like something RSMP should probably fix, but adding !RSMP to the relevant patches on our end probably won't hurt.

**Comment by linuxgurugamer** [comment]
Yes, that's what I did, but if I have to do an update or reinstall (made 5 installs working on a new game), it's just a pain

**Comment by Rodg88** [comment]
Ah there's no chance of this getting fixed on RSMP's side, it's basically been abandoned in favor of Avalanche Plumes - https://forum.kerbalspaceprogram.com/topic/207196-112x-rsmp-formally-swe/?do=findComment&comment=4404502
So I'll definitely add the fix on our end, in case people still happen to use it.

**Comment by Rodg88** [comment]
Ah this was already fixed in dev in 33e6159
